p2p/chain.py

```diff
@@ -304,10 +304,14 @@ class LightChainSyncer(BaseHeaderChainSyncer):
 
     async def _process_headers(
             self, peer: HeaderRequestingPeer, headers: Tuple[BlockHeader, ...]) -> int:
+        start = time.time()
```

Comment:
What do you think about writing a simple utility for this.

```
with timer() as stopwatch:
    ...  # do things
self.logger.info("thing took %d", stopwatch.elapsed)

# Or explicit start/stop
stopwatch = timer()
self.logger.info("thing took %d", stopwatch.elapsed)
```

I think there are multiple places in our code where we coul

Reply:
We have something like that already.

https://github.com/ethereum/py-evm/blob/462ae35cbfa8bede00baeec1db92bdb720fede2b/scripts/benchmark/utils/meters.py#L11-L20

Reply:
Added one under p2p/ as at this time we only seem to need that functionality there.

Reply:
`TimedResult` doesn't seem like a good fit here as in the p2p code we're usually timing just a few statements. And in one instance (`StateSyncer`) we actually want to keep the start time it as an instance attribute so that we can use it on the stats we report periodically